js/sts-hotspot.js

```diff
@@ -0,0 +1,45 @@
+AFRAME.registerComponent('sts-hotspot', {
+  schema: {
+    show: { type: 'selector' },
+    sound: { type: 'selector' }
+  },
+
+  init: function () {
+    var self = this;
+
+    self.el.addEventListener('mouseenter', function (e) {
+      self.playHotspotSound();
+
+      // make popup visible
+      console.log('showing:',self.data.show);
+      self.data.show.setAttribute('material', { visible: true });
+      self.data.show.setAttribute('visible', true);
+    })
+  },
+
+  playHotspotSound: function () {
+    if (!this.data.sound) {
+      return;
+    }
+    this.stopHotspotSound();
+
+    // play hotspot audio
+    this.data.sound.load();
+    this.data.sound.play();
+
+    this.el.dataset.playing = true;
+  },
+
+  stopHotspotSound: function () {
+    var playing = document.querySelector('[data-playing]');
```

Comment:
`playing` -> `playingEl`

Reply:
I'm not sure how, but one time I was able to get two sounds playing simultaneously. the sounds were indecipherable.

perhaps this should this be `document.querySelectorAll('[data-playing]')`? and then in a `forEach` loop, pause any playing sound and remove the 'data-playing' attribute.